jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BaseImageParameters.java

```diff
@@ -29,13 +30,30 @@
 public class BaseImageParameters {
 
   private final AuthParameters auth;
-
-  @Nullable private String image;
   @Nullable private String credHelper;
+  @Nullable private String image;
+  private final PlatformsParametersSpec platformsParametersSpec;
+  private ListProperty<PlatformsParameters> platforms;
```

Comment:
I guess this can be `final`. (`final` in Java means the reference `platforms` must be assigned when the class is instantiated and cannot be changed to something else later.)
```suggestion
  private final ListProperty<PlatformsParameters> platforms;
```